src/analyze/analyze.c

```diff
@@ -66,7 +66,7 @@ static enum dot {
 static char** arg_dot_from_patterns = NULL;
 static char** arg_dot_to_patterns = NULL;
 static usec_t arg_fuzz = 0;
-static bool arg_no_pager = false;
+static bool arg_pager = 0;
```

Comment:
bool → PagerFlags

Reply:
I think it would be nicer to call this arg_pager_flags, so that people don#t accidentally start using this as bool